csrc/mmdeploy/core/registry.cpp

```diff
@@ -78,4 +78,4 @@ Span<Creator<void>*> Registry<void>::Creators() {
 
 }  // namespace _registry
 
-}  // namespace mmdeploy
+}  // namespace mmdeploy
```

Comment:
may recover to the original.

Reply:
copied this file from master, don't where this difference comes from, looks the same to me

Reply:
lacks a blank line. Github needs a blank line in the end of a file to view.